src/app/codegen/codeGenerator.ts

```diff
@@ -187,37 +175,45 @@ function generateModel(graph: Graph, name: string): string {
   return [header, initMethod, forwardMethod].join('\n\n');
 }
 
-function generateFunctions(graph: Graph): string {
-  const customNodes = graph.nodesAsArray.filter((item: GraphNode) => item.mlNode instanceof Custom);
+function importCustomFunctions(graph: Graph): string[] {
+  const customNodes = graph.nodesAsArray.filter((item: GraphNode) => item.mlNode instanceof Custom || item.mlNode instanceof OverviewCustom);
```

Comment:
This will need to be `item.mlNode instanceof ModelCustom || item.mlNode instanceof DataCustom || item.mlNode instanceof OverviewCustom`